app/scripts/models/auth_brokers/base.js

```diff
@@ -481,6 +481,11 @@ define(function (require, exports, module) {
        * Should the view handle signed-in notifications from other tabs?
        */
       handleSignedInNotification: true,
+      /**
+       * If the user has an existing sessionToken, can we safely re-use it
+       * on subsequent signin attempts rather than generating a new token each time?
+       */
+      reuseExistingSession: false,
```

Comment:
FWIW, this matches my understanding of brokers in the repo (although I could be wrong).